images/bay-php/bay-shared-temp-files-dir-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
This is a nit, and honestly could go either way https://www.baeldung.com/linux/bash-shebang-lines#comparison

Reply:
Leaving this as is because it's in line with the [uselagoon/lagoon entrypoint scripts](https://github.com/uselagoon/lagoon-images/blob/main/images/commons/lagoon/entrypoints/999-readiness.sh)